PowerEditor/src/Parameters.cpp

```diff
@@ -149,6 +149,7 @@ static const WinMenuKeyDefinition winKeyDefs[] =
 	{ VK_SPACE,   IDM_EDIT_AUTOCOMPLETE_PATH,                   true,  true,  false, nullptr },
 	{ VK_RETURN,  IDM_EDIT_AUTOCOMPLETE_CURRENTFILE,            true,  false, false, nullptr },
 	{ VK_SPACE,   IDM_EDIT_FUNCCALLTIP,                         true,  false, true,  nullptr },
+	{ VK_NULL,    IDM_EDIT_FUNCCALLTIP_NEXT,                    false, false, false, nullptr },

```

Comment:
I'm wondering if we should add Alt-Down & Alt-Up by default.
What do you people think?

Reply:
> I'm wondering if we should add Alt-Down & Alt-Up by default.
> What do you people think?

I prefer no defaults and let people create their own.  The other thought is @alankilborn 's comment about somehow making this "just work" with `UP`/`DOWN` arrows themselves when a calltip window is active, like `UP`/`DOWN` arrows just work on AutoComplete combo boxes when they appear.  I don't know how / if this is possible, but maybe worth looking at?

Cheers.